event-svc/src/event/service.rs

```diff
@@ -403,7 +403,7 @@ impl EventService {
                         .map_err(|e| {
                             Error::new_app(anyhow::anyhow!("Failed to serialize IPLD data: {}", e))
                         })?,
-                    index: 0,
+                    index: delivered as u64,
```

Comment:
!!

Reply:
Tests should have caught this

Reply:
No test included an unsigned init event. That is why I added it to the generated data.